unskript-ctl/add_notification.py

```diff
@@ -27,11 +28,15 @@
 # Note, the earlier name we used was unskript_config.yaml. 
 if os.path.exists('/unskript/data/action/unskript_config.yaml') is True:
     try:
-        os.makedirs(Path(GLOBAL_CONFIG_PATH).parent, exist_ok=True)
+        os.makedirs(Path(GLOBAL_CONFIG_PATH).parent, exist_ok=True) 
         Path('/unskript/data/action/unskript_config.yaml').rename(GLOBAL_CONFIG_PATH)
     except:
         pass
 
+# Lets ensure the Global Config file exists in the
+# path
+if os.path.exists(GLOBAL_CONFIG_PATH) is False:
+    Path(GLOBAL_CONFIG_PATH).touch()
```

Comment:
How is this file supposed to be created? 
Is this a hack to touch this file ? It should probably just be there in the DockerFile as a step rather than in python

Reply:
Yes, that make sense, will package it in the DockerFile itself.

Reply:
Done. Will leave this in just as a safety net